src/libConsensus/ConsensusLeader.cpp

```diff
@@ -629,13 +629,6 @@ bool ConsensusLeader::ProcessMessageResponseCore(
       // Multicast to all nodes in the committee
       // =======================================
 
-      // FIXME: quick fix: 0106'08' comes to the backup ealier than 0106'04'
```

Comment:
No longer relevant, since we allow backup to receive finalcollectivesig before collectivesig